### DISCUSSION TITLE
애니메이션 제공 질문

### DISCUSSION THREAD

**Discussion by yoonjoohwan** [naverz/zepeto-studio-kor#600] [5. Q&A]
혹시 제페토 내부적으로 만들어준 애니메이션은 package 내부에 있는 base 들 말고 없는 것인가요?
혹시 있다면 그 애니메이션들을 제공 받을 수 있을까요?

**Comment by ted002** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/538

애니메이션 관련은 이쪽을 확인해보시면 좋을듯 합니다!
